src/lib/viewers/controls/annotations/annotations-button-region-tooltips/AnnotationsButtonRegionTooltips.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useContext, PropsWithChildren } from 'react';
+
+import { ExperiencesContext } from '../../experiences';
+
+import AnnotationsFlowTooltip from './AnnotationsFlowTooltip';
+import PersistentOnboardingBoxEditAnnotationsTooltip from './PersistentOnboardingBoxEditAnnotationsTooltip';
+
+export type Props = PropsWithChildren<{
+    isEnabled?: boolean;
+}>;
+
+export default function AnnotationsButtonRegionTooltips({ children, isEnabled }: Props): JSX.Element {
+    const { experiences } = useContext(ExperiencesContext);
+
+    const { persistentOnboardingBoxEditAnnotations: experience } = experiences;
+    const canShow = experience ? experience.canShow : false;
+
+    if (canShow) {
```

Comment:
I'm concerned about this conditionally rendering tooltip, the useTargetingApi will display the tooltip accordingly based on the canShow property and they will never both be true at the same time if they are wrapped correctly in EUA in the makeUseShowOne hook. We've seen problems in the past were the methods in the targeting api aren't called correctly or the children component event listeners aren't called correctly when conditionally rendering tooltips. Can we instead leave the original `AnnotationsTargetedTooltip` as it is and have nested tooltips?

Reply:
@robertjk - I wonder if it's possible to use just one component and modify props? I find there is the double click issue with the annotations tooltip. I also notice another problem where `isDiscoverabilityEnabled` isn't being updated after the initial page load. Here are some videos to demonstrate the issues:

1. https://cloud.box.com/s/turwgqzglk2x2gbbo6c8syklt7zww3jv - you will notice that I click next on the preview open button tooltip before the control bar loads, therefore when it loads, canShow is true for the annotations tooltip, and `isDiscoverabilityEnabled` is properly returning false so there is no default selected. When I click on the annotations region button, it closes the tooltip but doesn't select the region button, therefore you need to click it twice (this is the bug that always happens when conditionally rendering tooltips like this). Therefore, I believe we need to use the same tooltip component and modify props if we want to avoid this issue. And also, we should update `isDiscoverabilityEnabled` to change when experiences canShow value changes, since this tooltip doesn't typically appear on page load.

2. https://cloud.box.com/s/vy87li6yoxqfy9qiz9v17xtb1qp3ib8j - you will notice if I wait for the control bar to load and I don't click next to view the annotations tooltip, the annotations region will be default selected. When I click next and the annotations tooltip appears, it doesn't change, even though canShow is now true. When I click on the button, the tooltip closes and it doesn't deselect. (In this case the bug isn't bad because we still have the annotations region selected) However, this is not the expected behavior.